### ISSUE TITLE
ability to change the order of volume buttons

### ISSUE THREAD

**Issue by BossToroech** [KotatsuApp/kotatsu-parsers#707]
### Describe your suggested feature

I think there should be an an option to reverse the order of volume buttons to avoid potential problems with control.

### Other details

Hi,
I have a little problem with controlling using the volume keys.  I have wired headphones with volume buttons connected, but when I press "-" to move forward, the reader first goes back and only after the second click moves forward.  The "+" button does not have this problem. I think it's the fault of the headphones and the way they function, because I had the same thing in the Tachiyomi application, but there was an option to reverse the buttons, witch solved the problem.  Would there be an option here to reverse these buttons to avoid similar problems?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by ztimms73** [comment]
you opened the issue in a wrong repo
